lms/templates/seq_module.html

```diff
@@ -44,10 +44,11 @@
      ${item['content'] | h}
   </div>
   % endfor
-  <div id="seq_content" role="tabpanel"></div>
+  <div id="sr-is-focusable" tabindex="-1"></div>
```

Comment:
@talbs @clrux, my concern with using sr is that it would be offscreen, and we *may* get a weird focus indicator that goes offscreen when someone clicks on a tab.  Since it is not visible anyway (It's empty) we don't need to move it offscreen with sr class.  We need an ID so that we can programmatically identify it with javascript with focus().  I think something short and sweet should work i.e. id="sr-focus"  as in, this is just here to move focus for screen reader users.